### PR TITLE
On delete cascade from errors table

### PR DESCRIPTION
**Pull request by tisnik**
# Description

On delete cascade from errors table

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Unit tests (no changes in the code)

## Testing steps

Done on CI

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: mig_0006_on_delete_cascade_from_errors_table.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+// Generated documentation is available at:
+// https://pkg.go.dev/github.com/RedHatInsights/ccx-notification-writer/
+//
+// Documentation in literate-programming-style is available at:
+// https://redhatinsights.github.io/ccx-notification-writer/packages/mig_0005_create_read_errors_table.html
```
Comment (epapbak) [inline mig_0006_on_delete_cascade_from_errors_table.go:21]: 0006

**Review by epapbak — APPROVED** [review]
Thanks!

File: mig_0006_on_delete_cascade_from_errors_table.go (hunk 1 above)
Comment (tisnik) [reply mig_0006_on_delete_cascade_from_errors_table.go]: Fixed

File: mig_0006_on_delete_cascade_from_errors_table.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+// mig0006OnCascadeDeleteFromErrorsTable migration creates table named `event_targets`.
+// This table contains all event receivers, for example Notification Backend
+// and ServiceLog
+var mig0006OnCascadeDeleteFromErrorsTable = mig.Migration{
+	StepUp: func(tx *sql.Tx, _ types.DBDriver) error {
+		log.Debug().Msg("Executing mig0006OnCascadeDeleteFromErrorsTable stepUp function")
+		query := `
+		    ALTER TABLE read_errors
+		    DROP CONSTRAINT read_errors_org_id_fkey,
+		    ADD CONSTRAINT  read_errors_org_id_fkey
```
Comment (Bee-lee) [inline mig_0006_on_delete_cascade_from_errors_table.go:39]: nice, TIL these don't have to be done in a separate transaction

**Review by Bee-lee — APPROVED** [review]

File: mig_0006_on_delete_cascade_from_errors_table.go (hunk 2 above)
Comment (tisnik) [reply mig_0006_on_delete_cascade_from_errors_table.go]: yup ;)

**Pull request merged**